sdk/servicebus/azservicebus/client.go

```diff
@@ -73,8 +73,8 @@ func (client *Client) NewProcessor(options ...ProcessorOption) (*Processor, erro
 }
 
 // NewSender creates a Sender, which allows you to send messages or schedule messages.
-func (client *Client) NewSender(queueOrTopic string, options ...SenderOption) (*Sender, error) {
-	sender, err := newSender(client.namespace, queueOrTopic, options...)
+func (client *Client) NewSender(queueOrTopic string) (*Sender, error) {
```

Comment:
Why were the options removed?

Reply:
The only option that existed was one to set the sessionID for a Sender.

In track 2 there isn't a session association with a sender (it's expected to be set explicitly in each message that's sent).

Interestingly enough (if you follow the rabbit hole a bit more) there are some spots that accepted sender options but then didn't propagate them to the underlying call. So it was spotty at best (maybe on purpose...)